src/main/java/seedu/command/DeadAddTranctionCommand.java

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
Similarly, this entire file has been commented out. You can consider removing this,